### DISCUSSION TITLE
Generic targets missing in Configurator?

### DISCUSSION THREAD

**Discussion by fiechr** [ExpressLRS/ExpressLRS#2726] [Q&A] [answered]
A few weeks ago, I was installing ELRS on a "Cyclone" named RX. I remember using a generic target for 6 or 7 PWM channels, but I can't find this entry anymore (even for older versions)? Is this a bug or on purpose?

This is a screenshot from a YT video (not mine) that shows how it looked before:
![genericTargetsMissing](https://github.com/ExpressLRS/ExpressLRS/assets/12040376/163faa22-b6a8-40e0-8f32-ea9309a4cffb)

**Comment by pkendall64** [comment, marked as answer]
This was a decision from the dev team due to manufacturers not getting involved in the project and users running into issues with hardware that has not been tested.

**Comment by fiechr** [comment]
Are there any official recommendations what to do in this case?

It's about this receiver:

![Cyclone7CHELRSRX_Front](https://github.com/ExpressLRS/ExpressLRS/assets/12040376/2f18009e-0828-4bc2-8498-9e6de01b181d)
![Cyclone7CHELRSRX_Back](https://github.com/ExpressLRS/ExpressLRS/assets/12040376/29ead296-6d92-4742-b682-8ed3b1e1b41e)

AFA I can tell the receiver worked fine with the 6xPWM firmware (but not with the 7xPWM firmware, as the marking on the PCB may suggest).
I don't know who the manufacturer is and I can't tell if this thing works in every aspect possible. Is there some kind of "use at your own risk" category?

**Reply by 1nformatica** [reply]
I show how to use channel 7 in this video : https://youtu.be/2VZb0n_G_Pg Now the generic / DIY options have been removed I used the Happymodel 6 PWM target successfully.

**Reply by JBFUK** [reply]
So what's the correct way of building/flashing a generic target now?  Appreciate it may not be via the configurator but there must still be a way to flash a generic target?

**Comment by samsab1** [comment]
hello all
i have flashed my cyclone with happymodel 6 pwm last version 3.5.3
receiver works, but i have very low servo output travel
do you know what can be the cause?
best regards
